### ISSUE TITLE
400 Error when requesting balance: "Expected a JSON payload with accounts" (Gemini)

### ISSUE THREAD

**Issue by willrandship** [ccxt/ccxt#9085]
I was just trying to get started, and spent a fair amount of time looking at the manual for this, but I can't find anything documenting parameters for "fetchBalance". The gemini exchange definition seems to want something in the params, but I can't figure out how to make it happy.

I've provided a small code snippet below that recreates the issue with verbose logging enabled.
Any help would be appreciated.

- OS: Arch Linux
- Programming Language version: Python 3.9.1
- CCXT version: 1.49.3
- Exchange: Gemini

Code:
```
import ccxt
import secrets

gemini = ccxt.gemini({
    'apiKey': secrets.gemini_key,
    'secret': secrets.gemini_secret
})
gemini.verbose = True
gemini.fetchBalance()

```
Output:
```
Request: POST https://api.gemini.com/v1/balances {'Content-Type': 'text/plain', 'X-GEMINI-APIKEY': '<censored>', 'X-GEMINI-PAYLOAD': 'eyJyZXF1ZXN0IjoiL3YxL2JhbGFuY2VzIiwibm9uY2UiOjE2MTk1OTY1NTI3NDZ9', 'X-GEMINI-SIGNATURE': 'f51a41e420dc28971155e69c672322700930378c646206ea1e6aac53ede17c9819a55e8a7c27381c3e2826526843561b', 'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate'} None

Response: POST https://api.gemini.com/v1/balances 400 {'Date': 'Wed, 28 Apr 2021 07:55:52 GMT', 'Content-Type': 'application/json', 'Content-Length': '95', 'Connection': 'keep-alive', 'Server': 'nginx', 'Vary': 'Origin'} {"result":"error","reason":"MissingAccounts","message":"Expected a JSON payload with accounts"}
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ccxt/base/exchange.py", line 606, in fetch
    response.raise_for_status()
  File "/usr/lib/python3.9/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://api.gemini.com/v1/balances

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/site-packages/ccxt/gemini.py", line 596, in fetch_balance
    response = self.privatePostV1Balances(params)
  File "/usr/lib/python3.9/site-packages/ccxt/base/exchange.py", line 474, in inner
    return entry(_self, **inner_kwargs)
  File "/usr/lib/python3.9/site-packages/ccxt/base/exchange.py", line 499, in request
    return self.fetch2(path, api, method, params, headers, body)
  File "/usr/lib/python3.9/site-packages/ccxt/base/exchange.py", line 495, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/usr/lib/python3.9/site-packages/ccxt/base/exchange.py", line 622, in fetch
    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)
  File "/usr/lib/python3.9/site-packages/ccxt/gemini.py", line 821, in handle_errors
    raise ExchangeError(feedback)  # unknown message
ccxt.base.errors.ExchangeError: gemini Expected a JSON payload with accounts
```

**Comment by willrandship** [comment]
I figured it out.

Gemini has two types of API keys, account keys and master keys. If you use a master key, it requires you to specify which account you're referring to, even if there is only one.

I changed the key out for an account key, and now it works. Might be worth adding to the docs, though, or perhaps checking that an account key has been provided instead of a master key. Account keys start with 'account-' vs 'master-', so it would be as easy as

if 'account' not in apiKey:
  print('An invalid api key was provided. Note that you must use an account-scoped API key, as master keys are unsupported.')
